e2e/internal/contrasttest/contrasttest.go

```diff
@@ -96,6 +96,7 @@ func (ct *ContrastTest) Init(t *testing.T, resources []any) {
 	resources = kuberesource.PatchImages(resources, ct.ImageReplacements)
 	resources = kuberesource.PatchNamespaces(resources, ct.Namespace)
 	resources = kuberesource.PatchServiceMeshAdminInterface(resources, 9901)
+	resources = kuberesource.PatchCoordinatorMetrics(resources, 9102)
```

Comment:
Not sure about patching this within `contrasttest`. We might want to add a test case where we disable the metrics endpoint to check it is actually not reachable. Same for the envoy admin iface or other config things. Any thoughts on this @burgerdev ?

Anyway, nothing we must address in this PR.

Reply:
I agree: we should eventually test these scenarios, but until then we can leave it here.